packages/grafana-ui/src/components/Graph/Graph.tsx

```diff
@@ -79,22 +79,26 @@ export class Graph extends PureComponent<GraphProps> {
     const ticks = width / 100;
     const min = timeRange.from.valueOf();
     const max = timeRange.to.valueOf();
-    const yaxes = uniqBy(
-      series.map(s => {
-        const index = s.yAxis ? s.yAxis.index : 1;
-        const min = s.yAxis && !isNaN(s.yAxis.min as number) ? s.yAxis.min : null;
-        const tickDecimals = s.yAxis && !isNaN(s.yAxis.tickDecimals as number) ? s.yAxis.tickDecimals : null;
-
-        return {
-          show: true,
-          index,
-          position: index === 1 ? 'left' : 'right',
-          min,
-          tickDecimals,
-        };
-      }),
-      yAxisConfig => yAxisConfig.index
-    );
+
+    const noDataToBeDisplayed = series.length === 0;
+    const yaxes = noDataToBeDisplayed
+      ? [{ show: true, min: -1, max: 1 }]
+      : uniqBy(
```

Comment:
to make this more readable would break this out into functions so we can get rid of the multi-line ? expression.

First a function getYAxes(...) 

first in that function we could do
if (series.length === 0) {
return [{ show: true, min: -1, max: 1 }]
}

Reply:
Refactored. Thanks for feedback @torkelo! 🔝